python/ray/autoscaler/v2/instance_manager/storage.py

```diff
@@ -0,0 +1,67 @@
+import abc
+import copy
+from abc import abstractmethod
+from threading import Lock
+from typing import Dict, List, Optional, Set, Tuple
+
+
+class Storage(metaclass=abc.ABCMeta):
+    @abstractmethod
+    def update(
+        self,
+        table: str,
+        mutation: Dict[str, str],
```

Comment:
i wonder if we could change the Storage interface to take typed protobuf rather than strings? 
I assume string is needed if we have external storage that's not aware of the protobuf type, but I guess serialization could happen there? There isn't a need for InMemoryStorage to use string right?